go/obscuronode/host/host.go

```diff
@@ -51,7 +55,7 @@ type Node struct {
 	ID common.Address
 
 	P2p          P2P
-	ethereumNode obscurocommon.L1Node
+	ethConnector l1client.EthereumClient
```

Comment:
@otherview I think we either rename the type, or we rename the field, to be consistent. It's confusing to refer to it as a connector here, but a client in the type name.

Reply:
agree. let's name it consistently. Either with Joel's suggestion or name the class "EthConnector"

Reply:
Good point - Changing it to EthClient as suggested below